packages/grafana-ui/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -0,0 +1,43 @@
+import { css, keyframes } from '@emotion/css';
+import React from 'react';
+
+/**
+ * @internal
+ */
+export interface LoadingBarProps {
+  containerWidth: number;
+  width?: number;
+  height?: number;
+  ariaLabel?: string;
+}
+
+/**
+ * @internal
+ */
+export const LoadingBar: React.FC<LoadingBarProps> = ({ containerWidth, width, height, ariaLabel = 'Loading bar' }) => {
```

Comment:
I think we can just use a regular function definition (or a fat-arrow like here) without `React.FC` to follow [our guidelines](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components).

```suggestion
export function LoadingBar ({ containerWidth, width, height, ariaLabel = 'Loading bar' }: LoadingBarProps) {
```

Reply:
Yes! Thank you